events/wc_class_events.txt

```diff
@@ -913,94 +913,111 @@ character_event = {
 	option = {
 		name = EVTOPTA_WCCLS_110
 		random_list = {
-			60 = {
-				modifier = {
-					factor = 2
-					has_5_6_physical_class_or_higher_trigger = yes
-				}
+			200 = {
 				modifier = {
 					factor = 2
 					has_7_8_physical_class_or_higher_trigger = yes
 				}
 				modifier = {
 					factor = 2
 					has_9_10_physical_class_trigger = yes
-				}			
+				}
+
 				add_character_modifier = { name = beast_tortoise duration = -1 }
 			}
-			30 = { add_trait_wounded_effect = yes }
-			10 = { add_maimed_trait_effect = yes }
+			25 = { add_trait_wounded_effect = yes }
+			10 = {
+				trigger = { has_5_6_physical_class_or_higher_trigger = no }
+				add_maimed_trait_effect = yes
+			}
+			5 = {
+				trigger = { has_3_4_physical_class_or_higher_trigger = no }
+				death = { death_reason = death_accident_generic_beast }
+			}
```

Comment:
So, the cost of a tortoise is the same as the cost of a bear though you have more chances to die if you tame a bear.

Reply:
How about now?

Reply:
Looks fine.